conf/prod.routes

```diff
@@ -1,5 +1,7 @@
 # Add all the application routes to the app.routes file
-->         /individuals-income-api              app.Routes
-->         /                          health.Routes
+->         /individuals-income-api        app.Routes
```

Comment:
I seem to remember this inclusion of the service name in `/individuals-income-api` causing issues and I think it should just be `/`. Cannot remember what the exact issue was but it may have been an issue with publishing.

Reply:
I think you're right. I'll have a look.